streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/BaseRayClusterTest.java

```diff
@@ -0,0 +1,20 @@
+package io.ray.streaming.runtime;
+
+import io.ray.api.Ray;
+import java.lang.reflect.Method;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseRayClusterTest extends BaseUnitTest {
```

Comment:
Do we need to add a single lock for cluster test to ensure ray cluster is not started concurrent?ly